### ISSUE TITLE
Errors following tutorial

### ISSUE THREAD

**Issue by cormullion** [quarto-dev/quarto-web#289]

**Comment by dragonstyle** [comment]
Good suggestion. For your current issue has that been solved? If not, could you share the document that is causing this issue?

**Comment by cormullion** [comment]
Yes, I went back and redid all the installation commands with the `--user` option, and have now succeeded in rendering Quarto documents. Perhaps this issue boils down to working effectively with the  Apple/MacOS 'security enhancements' in Monterey... :)
